### ISSUE TITLE
When changing end time of series recording all scheduled recordings are gone

### ISSUE THREAD

**Issue by cesalo** [jellyfin/jellyfin#1994]
**Describe the bug**
I've been using the DVR functionality for over a month with no issues. Today I wanted to extend 5 minutes since I was missing the last part of one the shows i DVR and then noticed that all the schedule of the show was gone. Initially wasn't sure what caused that but then after retracing my steps I confirmed it by testing the same change in different shows.
I'm in version 10.4.0 using docker.
**To Reproduce**
1. Go to Live TV / Guide and pick a show
2. Click on the show and then on Record Series
3. Verify schedule
  - Go to Live TV/Series
  - Click on the show, scroll down and check the red icons.
4. In the same location as #3 change "Stop when possible" and add 5 minutes
5. Go back to Series and click on the same show, scroll down all episode are now with a grey icon.

**Expected behavior**
That changing the end time will not un-schedule the episodes of the show.
**Logs**
There are no errors but in the logs I can see the changes in the times:
```
[2019-11-08 19:36:56.511 +00:00] [INF] Creating recording timer for "8c27cf3460cf5d8572f41327879eaee3", "Erin Burnett OutFront". Timer will fire in "263.058151033333" minutes
[2019-11-08 19:36:56.512 +00:00] [INF] Creating recording timer for "8c27cf3460cf5d8572f41327879eaee3", "Erin Burnett OutFront". Timer will fire in "263.0581331" minutes
[2019-11-08 19:36:56.513 +00:00] [INF] Creating recording timer for "cd70f5287cbeef7e62ab8d4b9edf2817", "Erin Burnett OutFront". Timer will fire in "4583.05811707167" minutes
[2019-11-08 19:36:56.513 +00:00] [INF] Creating recording timer for "cd70f5287cbeef7e62ab8d4b9edf2817", "Erin Burnett OutFront". Timer will fire in "4583.05810241833" minutes
[2019-11-08 19:36:56.514 +00:00] [INF] Creating recording timer for "979561c37d77042ce79794fb3a49964c", "Erin Burnett OutFront". Timer will fire in "6023.058087125" minutes
[2019-11-08 19:36:56.515 +00:00] [INF] Creating recording timer for "979561c37d77042ce79794fb3a49964c", "Erin Burnett OutFront". Timer will fire in "6023.05806986833" minutes
[2019-11-08 19:36:56.516 +00:00] [INF] Creating recording timer for "0a04b4b663abd256d78da8fca68341aa", "Erin Burnett OutFront". Timer will fire in "7463.05805412167" minutes
[2019-11-08 19:36:56.517 +00:00] [INF] Creating recording timer for "0a04b4b663abd256d78da8fca68341aa", "Erin Burnett OutFront". Timer will fire in "7463.05803873667" minutes
[2019-11-08 19:36:56.518 +00:00] [INF] Creating recording timer for "2f87b9dddea9591116000bf5a8985cfe", "Erin Burnett OutFront". Timer will fire in "8903.05802003167" minutes
[2019-11-08 19:36:56.519 +00:00] [INF] Creating recording timer for "2f87b9dddea9591116000bf5a8985cfe", "Erin Burnett OutFront". Timer will fire in "8903.058002835" minutes
```

**Comment by cesalo** [comment]
Confirmed that this is not related with the baseurl issue. I removed the baseurl and still had the same issue.

**Comment by sudotm** [comment]
This issue is affecting me on the latest stable version of Jellyfin, 10.6.2, installed with the official debian package.

In my case, I set the "start before" time to 1 minute, then the recording disappears from my schedule and all future episodes are no longer recorded. This only occurs when adjusting the start/end times through _Series settings_ and does not occur when a single episode's start time is adjusted.

When I set the "1 minute before" record time in _Series settings_, the log shows that each request is passed twice for some reason. When I initially set the program to record, this duplication does not happen so it may be significant.

`[2020-08-26 21:32:33.456 -04:00] [INF] Creating recording timer for "07414001a6faca337599f5276bde4855", "Star Trek: Deep Space Nine". Timer will fire in "8666.442386831666" minutes
[2020-08-26 21:32:33.457 -04:00] [INF] Creating recording timer for "07414001a6faca337599f5276bde4855", "Star Trek: Deep Space Nine". Timer will fire in "8666.442372536667" minutes
[2020-08-26 21:32:33.458 -04:00] [INF] Creating recording timer for "815dc7f8ad3e5a3c7e05d3b1ebb88000", "Star Trek: Deep Space Nine". Timer will fire in "1466.4423584966667" minutes
[2020-08-26 21:32:33.459 -04:00] [INF] Creating recording timer for "815dc7f8ad3e5a3c7e05d3b1ebb88000", "Star Trek: Deep Space Nine". Timer will fire in "1466.4423444816666" minutes
[2020-08-26 21:32:33.460 -04:00] [INF] Creating recording timer for "8d7c707d1ab155e33c928ed11fe30ea5", "Star Trek: Deep Space Nine". Timer will fire in "2906.44232933" minutes
[2020-08-26 21:32:33.461 -04:00] [INF] Creating recording timer for "8d7c707d1ab155e33c928ed11fe30ea5", "Star Trek: Deep Space Nine". Timer will fire in "2906.442315011667" minutes`

I can go back into the info page for the episode and press the "Record" button, which will successfully reschedule the recording and inherit the "1 minute before" setting I set earlier. One time, when doing so, it raised an exception. This exception only occurs sometimes, it usually just says "New recording scheduled" without raising an exception.

`[2020-08-26 21:25:44.454 -04:00] [INF] Creating recording timer for "363a8042aae6dee8668cdd9f37f56c57", "Green Acres". Timer will fire in "1443.2590994433333" minutes
[2020-08-26 21:25:44.454 -04:00] [INF] New recording scheduled
[2020-08-26 21:25:44.587 -04:00] [ERR] Error processing request. URL: "http://jellyfin.site/LiveTv/Timers"
System.ArgumentException: A scheduled recording already exists for this program.
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.CreateTimer(TimerInfo timer, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.CreateTimer(TimerInfoDto timer, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
[2020-08-26 21:25:47.473 -04:00] [INF] Creating recording timer for "363a8042aae6dee8668cdd9f37f56c57", "Green Acres". Timer will fire in "1443.2087681183334" minutes`

I notice that the "New recording scheduled" message in the log never occurs when scheduling things as a series. I only see it in relation to actions on single episodes.
